Brightcoders-Bootcamps/kata-08-reloj-checador-team04-kata08
# Feature#10 front end

# Front End
 
## Description: Why did you change/implement that?
- Adding bootstrap, bootstrap-select-rails and bootstrap_form gem
- Adding styles for all pages in the app

## Testing: Help me how can I test or look at the changes?
1. Type `bundle install` in terminal for install gems of the project
2. Type `rails s` in terminal for initialize a rails server
- Go to **localhost:3000/admin** and login with _admin@example.com_ and _password_ to see the admin panel and explore it
- Go to **localhost:3000/sessions/login** and type any number in the input to see the messages the app shows
